manim/utils/space_ops.py

```diff
@@ -92,7 +92,7 @@ def rotate_vector(vector, angle, axis=OUT):
         product = reduce(quaternion_mult, [quat, np.append(0, vector), quat_inv])
         return product[1:]
     else:
-        raise Exception("vector must be of dimension 2 or 3")
+        raise ValueError("vector must be of dimension 2 or 3")
```

Comment:
Consider MathError?

Reply:
Well ……… it is not a math error as we are not trying to make something mathematically illegal, just something that this function can't handle
Plus, i can't find MathError ( i get MathError: not defined) on python nor in google. Does it realy exists in Python ?